diff --git a/dist/force-graph.js b/dist/force-graph.js
index fdc8e609384fc4c991073f1fc43b4cf322f88b3b..254128c3f41793e0914131bbf23c21153bfa54ca 100644
--- a/dist/force-graph.js
+++ b/dist/force-graph.js
@@ -10845,10 +10845,7 @@
           nodes: [],
           links: []
         },
-        onChange: function onChange(_, state) {
-          state.engineRunning = false; // Pause simulation
-          updDataPhotons(_, state);
-        }
+        triggerUpdate: false
       },
       dagMode: {
         onChange: function onChange(dagMode, state) {
diff --git a/dist/force-graph.mjs b/dist/force-graph.mjs
index 0fa712a82483ed98b0b9a65a7896ce66adeb8680..f00589f411d7aea8dfccaa1b8b931a1e0a4905f0 100644
--- a/dist/force-graph.mjs
+++ b/dist/force-graph.mjs
@@ -320,10 +320,7 @@ var CanvasForceGraph = Kapsule({
         nodes: [],
         links: []
       },
-      onChange: function onChange(_, state) {
-        state.engineRunning = false; // Pause simulation
-        updDataPhotons(_, state);
-      }
+      triggerUpdate: false
     },
     dagMode: {
       onChange: function onChange(dagMode, state) {
diff --git a/src/canvas-force-graph.js b/src/canvas-force-graph.js
index 0ef79176726eebe3aa58a5d2af14fd8b1c5a31fb..d4576a2d27eaf778afccfdf62a6c3c41cdfdb2a9 100644
--- a/src/canvas-force-graph.js
+++ b/src/canvas-force-graph.js
@@ -44,10 +44,7 @@ export default Kapsule({
         nodes: [],
         links: []
       },
-      onChange(_, state) {
-        state.engineRunning = false; // Pause simulation
-        updDataPhotons(_, state);
-      }
+      triggerUpdate: false
     },
     dagMode: { onChange(dagMode, state) { // td, bu, lr, rl, radialin, radialout
       !dagMode && (state.graphData.nodes || []).forEach(n => n.fx = n.fy = undefined); // unfix nodes when disabling dag mode
